.github/workflows/dapr.yml

```diff
@@ -71,6 +71,11 @@ jobs:
         with:
           name: dapr_${{ matrix.target_os }}_${{ matrix.target_arch }}
           path: ${{ env.ARCHIVE_OUTDIR }}
+      - name: upload grafana dashboards artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: dapr_grafana_dashboards
+          path: ./grafana/
```

Comment:
I thought we would want to upload it to github release. this uploads files to github actions build artifact.

Reply:
@youngbupark , When the release is cut,  wont this upload to github release ?

Reply:
Created the new workflow which will upload grafana dashboards to github release.